### DISCUSSION TITLE
Metrology is now in the Top 10! 🎉

### DISCUSSION THREAD

**Discussion by Madelena** [Madelena/Metrology-for-Hass#17] [Announcements]
2 weeks after its release, this theme is now in the Top 10 of the HACS themes section!

Thanks everyone for your support! 🥰 More features are coming!

Please share screenshots of how you are using it!

**Comment by marvandorth** [comment]
Hi Madelena,

I know you're probably busy with it already or have other tasks.
But I would love an headstart regarding your shared dashboard.
Like, how you created the grid and the button card templates etc.

**Reply by Madelena** [reply]
It's not quite done yet, but I had just posted an initial draft:

https://github.com/Madelena/hass-config-public/tree/documentation-init

Please let me know if you have questions, see something missing, or see something that should be private. 😀

**Reply by marvandorth** [reply]
> It's not quite done yet, but I had just posted an initial draft:
> 
> https://github.com/Madelena/hass-config-public/tree/documentation-init
> 
> Please let me know if you have questions, see something missing, or see something that should be private. 😀

Amazing news! :)
I'll get to it right away and let you know
